### PR TITLE
Highlight TiFlash does not support Lightning

### PR DESCRIPTION
**Pull request by flowbehappy**
### What is changed, added or deleted? (Required)

TiFlash does not support Lightning currently. We must explicitly notify the issue.

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v5.0 (TiDB 5.0 versions)
- [x] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs/pull/5119
- Other reference link(s):

### CODE REVIEW

File: tidb-lightning/tidb-lightning-overview.md [hunk 1]
```diff
@@ -44,3 +44,7 @@ TiDB Lightning 整体工作原理如下：
 如果需要导入的目标集群是 v3.x 或以下的版本，需要使用 Importer-backend 来完成数据的导入。在这个模式下，`tidb-lightning` 需要将解析的键值对通过 gRPC 发送给 `tikv-importer` 并由 `tikv-importer` 完成数据的导入。
 
 TiDB Lightning 还支持使用 TiDB-backend 作为后端导入数据：`tidb-lightning` 将数据转换为 `INSERT` 语句，然后直接在目标集群上执行这些语句。详见 [TiDB Lightning Backends](/tidb-lightning/tidb-lightning-backends.md)。
+
+> **注意：**
```
Comment (TomShawn) [inline tidb-lightning/tidb-lightning-overview.md:48]: 这个在 TiFlash 文档里说明会不会更好呢？

Comment (flowbehappy) [reply tidb-lightning/tidb-lightning-overview.md:48]: 我觉得用户的入口在 lightning 这边而不是在 tiflash，所以至少这里需要加一个提醒。

Comment (TomShawn) [reply tidb-lightning/tidb-lightning-overview.md:48]: 请将 https://github.com/pingcap/docs-cn/pull/5848 的改动内容提到这个 PR 中来吧。

File: tidb-lightning/tidb-lightning-overview.md [hunk 2]
```diff
@@ -44,3 +44,7 @@ TiDB Lightning 整体工作原理如下：
...
+>
+> TiDB Lightning 不支持对有 TiFlash 副本的表导入数据。在导入数据之前请先清除 TiFlash 副本。你可以在导入完成之后再添加 TiFlash 副本。
```
Comment (kennytm) [inline tidb-lightning/tidb-lightning-overview.md:50]: 🤔 if so this is likely a regression. we've got a [test case](https://github.com/pingcap/br/blob/master/tests/lightning_tiflash/run.sh) to ensure Lightning can work when TiFlash is present, but it seems it is skipped on the CI.

Anyway it should be clarified that this only affects the Local Backend.

Comment (flowbehappy) [reply tidb-lightning/tidb-lightning-overview.md]: My mistake. We can use Lightning with TiFlash after v4.0.6.

**Comment by flowbehappy** [comment]
@TomShawn PTAL

**Review by ilovesoup — APPROVED** [review]
lgtm

File: tidb-lightning/tidb-lightning-overview.md [hunk 3]
```diff
@@ -44,3 +44,8 @@ TiDB Lightning 整体工作原理如下：
 如果需要导入的目标集群是 v3.x 或以下的版本，需要使用 Importer-backend 来完成数据的导入。在这个模式下，`tidb-lightning` 需要将解析的键值对通过 gRPC 发送给 `tikv-importer` 并由 `tikv-importer` 完成数据的导入。
 
 TiDB Lightning 还支持使用 TiDB-backend 作为后端导入数据：`tidb-lightning` 将数据转换为 `INSERT` 语句，然后直接在目标集群上执行这些语句。详见 [TiDB Lightning Backends](/tidb-lightning/tidb-lightning-backends.md)。
+
+> **注意：**
+>
+> * 如果集群版本 \< v4.0.6，若先对表创建 TiFlash 副本，再使用 TiDB Lightning 导入数据，会导致数据导入失败。需要在使用 TiDB Lightning 成功导入数据至表后，再对相应的表创建 TiFlash 副本。
+> * 如果集群版本以及 TiDB Lightning 版本均 \>= v4.0.6，无论一个表是否已经创建 TiFlash 副本，你均可以使用 TiDB Lightning 导入数据至该表。但注意此情况会导致 TiDB Lightning 导入数据耗费的时间延长，具体取决于 TiDB Lightning 部署机器的网卡带宽、TiFlash 节点的 CPU 及磁盘负载、TiFlash 副本数等因素。
```
Comment (TomShawn) [inline tidb-lightning/tidb-lightning-overview.md:51]:
```suggestion
## 使用限制

+ TiDB Lightning 与 TiFlash 一起使用时需要注意：
    - 无论是否已为一张表创建 TiFlash 副本，你均可以使用 TiDB Lightning 导入数据至该表。但该场景下 TiDB Lightning 导入数据耗费的时间更长，具体取决于 TiDB Lightning 部署机器的网卡带宽、TiFlash 节点的 CPU 及磁盘负载、TiFlash 副本数等因素。
```

File: tidb-lightning/tidb-lightning-overview.md [hunk 4]
```diff
@@ -44,3 +44,10 @@ TiDB Lightning 整体工作原理如下：
 如果需要导入的目标集群是 v3.x 或以下的版本，需要使用 Importer-backend 来完成数据的导入。在这个模式下，`tidb-lightning` 需要将解析的键值对通过 gRPC 发送给 `tikv-importer` 并由 `tikv-importer` 完成数据的导入。
 
 TiDB Lightning 还支持使用 TiDB-backend 作为后端导入数据：`tidb-lightning` 将数据转换为 `INSERT` 语句，然后直接在目标集群上执行这些语句。详见 [TiDB Lightning Backends](/tidb-lightning/tidb-lightning-backends.md)。
+
+## 使用限制
+
+TiDB Lightning 与 TiFlash 一起使用时需要注意：
+
+无论是否已为一张表创建 TiFlash 副本，你均可以使用 TiDB Lightning 导入数据至该表。但该场景下 TiDB Lightning 导入数据耗费的时间更长，具体取决于 TiDB Lightning 部署机器的网卡带宽、TiFlash 节点的 CPU 及磁盘负载、TiFlash 副本数等因素。
+无论是否已为一张表创建 TiFlash 副本，你均可以使用 TiDB Lightning 导入数据至该表。但该场景下 TiDB Lightning 导入数据耗费的时间更长，具体取决于 TiDB Lightning 部署机器的网卡带宽、TiFlash 节点的 CPU 及磁盘负载、TiFlash 副本数等因素。
```
Comment (TomShawn) [inline tidb-lightning/tidb-lightning-overview.md:53]:
```suggestion
```

**Pull request merged**